### PR TITLE
Detailed Product Overview: Custom Upstreams

### PR DESCRIPTION
**Pull request by wordsmither**
Closes #DOCO-930

## Summary

**[Custom Upstreams](https://docs.pantheon.io/overview/custom-upstreams)** - New product overview page. One of many to come.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by wordsmither** [comment]
[Preview](https://pr-8249-documentation.appa.pantheon.site/overview/custom-upstreams)

File: source/content/overview/custom-upstreams.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+* [Custom Upstreams Guide](https://docs.pantheon.io/guides/custom-upstream)
+
+## Learning
+
+* [Custom Upstreams Training](https://learning.pantheon.io/page/catalog#topic_custom-upstreams)
+
+## Support
+
+* [Account holders](https://dashboard.pantheon.io/workspace/ee3995c4-652e-44a0-b00b-0085e92d78da/support)
```
Comment (jsulz) [inline source/content/overview/custom-upstreams.md:69]: Same note as on the New Relic review - this link 404s.

**Review by jsulz — CHANGES_REQUESTED** [review]
Looks correct; no issues with the content, just one requested change.

**Review by whitneymeredith — APPROVED** [review]
LGTM

**Pull request merged**